app/templates/homepage.html

```diff
@@ -15,6 +15,7 @@
           <nav class="section--hero__nav">
             <a href="#about">About</a>
             <a href="#login">Login</a>
+            <a href="#contact">Contact</a>
```

Comment:
I would keep homepage to tell about the application, about the purpose of the website.